modules/gatsby/package.json

```diff
@@ -33,7 +33,7 @@
     "gatsby": "^2.3.0",
     "react": "^16.6.0",
     "react-dom": "^16.6.0",
-    "sharp": "^0.22"
+    "sharp": "^0.22.0"
```

Comment:
Not sure if this makes much of a difference... Should we force it to 0.22.0 for now to make sure apps update their deps?

Reply:
done.